### PR TITLE
feat: pagination added in assets-reminders-company documents

### PR DESCRIPTION
**Pull request by FH-sandeepkaur1**
## Change Description

Changes made in following modules for pagination
1. Documents in routing and dispatch
2. assets in fleet
3. reminders in fleet

Listing fixes in routing module of routing and dispatch

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist: (Check only applicable)

- [y ] My code follows the style guidelines of this project
- [y ] I have performed a self-review of my own code
- [y ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [y ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by amritfleethawks — APPROVED** [review]

File: src/app/pages/fleet/assets/asset-list/asset-list.component.ts [hunk 1]
```diff
@@ -70,10 +82,11 @@ export class AssetListComponent implements OnInit {
...
 
-  ngOnInit() {
-      this.dataTableOptions();
+  ngOnInit(): void {
+      // this.dataTableOptions();
       this.fetchAssets();
       this.fetchAllAssetTypes();
+      this.initDataTable();
       // $(document).ready(() => {
       //   setTimeout(() => {
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/assets/asset-list/asset-list.component.ts:91]: Please remove the commented code.

File: src/app/pages/fleet/assets/asset-list/asset-list.component.ts [hunk 2]
```diff
@@ -109,143 +116,143 @@ export class AssetListComponent implements OnInit {
...
-            className: 'noVis'
-        }
-    ],
-    "fnDrawCallback": function(oSettings) {
-        if ($('.dataTables_wrapper tbody tr').length <= 10) {
-            $('.dataTables_paginate .previous, .dataTables_paginate .next').hide();
-        }
-    }
-    };
+  //   this.reeferOptions = { // Reefer list options
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/assets/asset-list/asset-list.component.ts:166]: Please remove the commented code.

File: src/app/pages/fleet/reminders/vehicle-renewals/vehicle-renew-list/vehicle-renew-list.component.ts [hunk 3]
```diff
@@ -82,65 +99,117 @@ export class VehicleRenewListComponent implements OnInit {
...
+  //           if (remainingDays < 0 ) {
+  //             reminderStatus = 'OVERDUE';
+  //           }
+  //           else if( remainingDays <= this.allRemindersData[j].reminderTasks.remindByDays &&  remainingDays >= 0) {
+  //             reminderStatus = 'DUE SOON';
+  //           }
+  //           const data = {
+  //             reminderID: this.allRemindersData[j].reminderID,
+  //             reminderIdentification: this.allRemindersData[j].reminderIdentification,
+  //             reminderTasks: {
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/reminders/vehicle-renewals/vehicle-renew-list/vehicle-renew-list.component.ts:123]: Please remove commented code.

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
@FH-sandeepkaur1 Please remove commented code.

**Review by kunalfleethawks — APPROVED** [review]

**Pull request merged**